lib/config.js

```diff
@@ -16,6 +16,12 @@ cfg.routes = {
   basePath
 };
 
+cfg.zcap = {
+  default: {
```

Comment:
I'm not sure we need the property default here. 

```js
cfg.zcap = {
    // default: 24 hour expiration
    ttl: 24 * 60 * 60 * 1000
};
```

Are there other libraries that use the property `default` in the `config.js`? 

@mattcollier

Reply:
@aljones15 we were using `defaults` in [here](https://github.com/digitalbazaar/bedrock-authn-token/pull/29/files). Since there's only one value here, I thought of calling it default. But now I am not sure either, I can remove it if necessary.

Reply:
If you look at that file nothing else has the property `default`: https://github.com/digitalbazaar/bedrock-authn-token/blob/master/lib/config.js

in this case the default value is 24 hours.